src/test/resources/misc/anarchism.xml

```diff
@@ -0,0 +1,453 @@
+<mediawiki xmlns="http://www.mediawiki.org/xml/export-0.10/" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.mediawiki.org/xml/export-0.10/ http://www.mediawiki.org/xml/export-0.10.xsd" version="0.10" xml:lang="en">
```

Comment:
Also, wouldn't it be more consistent to add just the page markup (without the top level xml stuff) into `resources/en` instead of `resources/misc` ?